pkg/debugapi/debugapi.go

```diff
@@ -30,33 +31,32 @@ import (
 
 type Service interface {
 	http.Handler
+	Configure(overlay swarm.Address, publicKey, pssPublicKey ecdsa.PublicKey, ethereumAddress common.Address, p2p p2p.DebugService, pingpong pingpong.Interface, topologyDriver topology.Driver, storer storage.Storer, tags *tags.Tags, accounting accounting.Interface, settlement settlement.Interface, chequebookEnabled bool, swap swap.ApiInterface, chequebook chequebook.Service)
```

Comment:
As explained in PR description, this interface should be removed for simplification and to avoid to have this method here.